src/utils/vcs.rs

```diff
@@ -355,14 +367,19 @@ pub fn find_head() -> Result<String, Error> {
 
 /// Given commit specs and repos this returns a list of head commits
 /// from it.
-pub fn find_heads(specs: Option<Vec<CommitSpec>>, repos: &[Repo]) -> Result<Vec<Ref>, Error> {
```

Comment:
The spec comment should be updated to mention remote as well.

Reply:
Comment line 368 now speaks about remote.